panic-itm/src/lib.rs

```diff
@@ -46,7 +46,7 @@ use cortex_m::peripheral::ITM;
 fn panic(info: &PanicInfo) -> ! {
     interrupt::disable();
 
-    let itm = unsafe { &mut *ITM::ptr() };
+    let itm = unsafe { &mut *ITM::PTR };
```

Comment:
ah whoops, this is in a different crate. How should we handle this? leave it to the master PR https://github.com/rust-embedded/cortex-m/pull/385/files?

Reply:
Hm, yea, put that in the #385 PR please, we can do a new panic-itm release from master.